libpod/image/image.go

```diff
@@ -1075,3 +1076,75 @@ func (i *Image) Comment(ctx context.Context, manifestType string) (string, error
 	}
 	return ociv1Img.History[0].Comment, nil
 }
+
+// Tree gets dependencies of image
+func (i *Image) Tree(ctx context.Context) ([]Image, error) {
+	treeDepth := 1
+	rootImage := i
+
+	//Traverse to the root(base) image
+	parent, err := i.GetParent()
+	if err != nil {
+		return nil, err
+	}
+
+	for parent != nil {
+		rootImage = parent
+		parent, err = parent.GetParent()
+		if err != nil {
+			break
+		}
+		treeDepth++
+	}
+
+	//Print all subtree(peers) to the depth of input image
+	printSubTree(rootImage, 0, "└─ ", treeDepth)
```

Comment:
Just to double check, are these standard ascii characters in the 3rd argument.  Will there be any trouble displaying this on a Mac or other non-linux device?

Reply:
ASCII is 0–127, this is not a part of it. But Golang source code is specified to be UTF-8, so it is unambiguous, and Unicode support has been good enough for about a decade.